src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/UserPersonDetail.kt

```diff
@@ -145,3 +150,36 @@ class CaseloadAlreadyExistsException(message: String?) :
     return CaseloadAlreadyExistsException(message)
   }
 }
+
+fun UserPersonDetail.toUserSummaryWithEmail() = UserSummaryWithEmail(
```

Comment:
wasn't a great fan of having a `Transformers` class that will just get bigger and bigger, so moved the summary stuff here